.github/workflows/deploy-maven.yaml

```diff
@@ -38,6 +38,6 @@ jobs:
 
       #Deploy  bpdm-common jar to GitHub Maven registry
       - name: Deploy
-        run: ./mvnw -pl bpdm-common deploy
+        run: ./mvnw -pl bpdm-common, bpdm-gate-api deploy
```

Comment:
bpdm-gate-api needs to have distribution management in its pom configured in order to be deployed